code/__HELPERS/areas.dm

```diff
@@ -61,12 +61,7 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(list(
 	var/list/connected_turfs = list(source)
 	. = connected_turfs
 	while(length(connected_turfs))
-		var/list/turf/adjacent_turfs = list(
-			get_step(connected_turfs[counter], NORTH),
-			get_step(connected_turfs[counter], SOUTH),
-			get_step(connected_turfs[counter], EAST),
-			get_step(connected_turfs[counter], WEST)
-		)// get a tile in each cardinal direction at once and add that to the list
+		var/list/turf/adjacent_turfs = source.atmos_adjacent_turfs // get a tile in each cardinal direction at once and add that to the list
```

Comment:
this is not valid, this means adjacent turfs is always just the same around the starting turf. why is this being changed and have you tested it

Reply:
aight my bad i thought i could do a quick fix